docs/portal/developer-portal/operations/Create_UAN_Boot_Images.md

```diff
@@ -247,8 +246,8 @@ See also [*HPE Cray EX System Software Stack Installation and Upgrade Guide for
 
 ```yaml
 session_templates:
-- name: uan-2.4.0
-  image: cray-shasta-uan-sles15sp3.x86_64-2.3.25
+- name: uan-2.7.3
+  image: cray-shasta-uan-sles15sp5.x86_64-2.7.3
```

Comment:
@keopp-hpe for 2.7.1, should the image line now say `image: cray-shasta-uan-sles15sp5.x86_64-2.7.1`

Reply:
Yes, for 2.7.1 it should reflect that.

Reply:
Just fixed it in the latest commit.